### PR TITLE
Check fullSetParametersFile for null before attempting a Path.Combine

### PR DESCRIPTION
**Pull request by eocrawford**
See title

### CODE REVIEW

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by hovsepm** [comment]
@eocrawford please fix build breaks

**Comment by eocrawford** [comment]
Hi, not sure what the issue is here or how to fix it. (My change contains a passing unit test.) I'm unable to access the Details URL (http://azuresdkci.cloudapp.net/job/powershell/4459/); fails with eocrawford is missing the Overall/Read permission. If you can grant that permission, I'll take a look at the details and see what I can do to fix the problem

**Comment by markcowl** [comment]
@eocrawford build failure, here are the errors from the log:
```
(CoreCompile target) -> 
17:16:42   Websites\PublishAzureWebsiteProjectTests.cs(25,53): error CS1525: Invalid expression term '[' [D:\workspace\powershell-2\src\ServiceManagement\Services\Commands.Test\Commands.Test.csproj]
17:16:42   Websites\PublishAzureWebsiteProjectTests.cs(25,54): error CS1003: Syntax error, ',' expected [D:\workspace\powershell-2\src\ServiceManagement\Services\Commands.Test\Commands.Test.csproj]
17:16:42   Websites\PublishAzureWebsiteProjectTests.cs(25,73): error CS1003: Syntax error, ',' expected [D:\workspace\powershell-2\src\ServiceManagement\Services\Commands.Test\Commands.Test.csproj]
```

File: src/ServiceManagement/Services/Commands.Test/Websites/PublishAzureWebsiteProjectTests.cs [hunk 1]
```diff
@@ -18,6 +17,57 @@ public class PublishAzureWebsiteProjectTests : WebsitesTestBase
     {
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void PublishFromPackageWithoutSetParameters()
+        {
+            const string websiteName = "test-site";
+            const string package = "test-package";
+            string slot = null;
+            var connectionStrings = new Hashtable { ["DefaultConnection"] = "test-connection-string" };
```
Comment (markcowl) [inline src/ServiceManagement/Services/Commands.Test/Websites/PublishAzureWebsiteProjectTests.cs]: @eocrawford This is where the compilation issue is, according to the log.

I believe you use the same literal construction as a dictionary, i.e.:

```c#
var connectionStrings = new HashTable {
    {"DefaultConnection", "test-connection-string"}
};
```

Comment (eocrawford) [inline src/ServiceManagement/Services/Commands.Test/Websites/PublishAzureWebsiteProjectTests.cs]: I've moved the hashtable initialization after the declaration to match the existing syntax in other test cases.

**Comment by markcowl** [comment]
@azuresdkci retest this please

**Comment by hovsepm** [comment]
@azuresdkci retest this please

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/620/  (soon)

**Comment by markcowl** [comment]
@eocrawford failure in the on demand run

**Comment by eocrawford** [comment]
I'm not able to view the results at http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/620/. How do I get permission? Alternately, you can paste the results here and I'll take a look.

**Comment by eocrawford** [comment]
I'm not able to repro the test failure locally and it happens outside the area I touched. Can we get a rerun?

**Comment by eocrawford** [comment]
@azuresdkci retest this please

**Comment by eocrawford** [comment]
@azuresdkci retest this please

**Comment by hovsepm** [comment]
on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/626/

**Pull request merged**